src/Lock.php

```diff
@@ -252,6 +269,14 @@ public static function showForItem(CommonDBTM $item)
         ]);
         $header |= $results['header'];
 
+        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
+        echo "<i class='fas fa-info-circle fa-xl'></i>";
+        echo "<span class='ms-2'>";
+        echo __("Here the locked items are displayed.<br> A locked items is a manually deleted items (example: a monitor).
```

Comment:
```suggestion
        echo __("A locked item is a manually deleted item (example: a monitor).
```

Reply:
Maybe should you use anther concatenated string for second sentence instead of using an html tag in translations.

Reply:
fixed